context-free.md

```diff
@@ -2,21 +2,42 @@
 
 ##  Who is this programming language for?
 
+This programming language is for people who want to create art on the computer 
+through rules. Instead of using a mouse to point and click like you would in 
+paint, you rather create mathematical rules that creates art.
 
 ## What is easy to do in this language? Why is it easy?
 
+It is easy to do recursive shapes. It is easy because the language allows you 
+to define rules with itself.
 
 ## What is hard to do in this language? Why is it hard?
 
+It is hard to do arbitrary shapes. If you wanted to draw letters, you have to 
+define multiple different and somewhat repetitive rules. It's nice to have the 
+render button to show you what you did immediately, but complex arbitrary 
+shapes still take a while. In the end this is because the rules do not cater 
```

Comment:
Why don't they cater to this? Anytime you run into something like this, your first instinct should be to look at the tradeoff, and who benefits from it. This will help a lot in analyzing a DSL and its target audience.